main/src/main/scala/sbt/Defaults.scala

```diff
@@ -3016,11 +3016,16 @@ trait BuildExtra extends BuildCommon with DefExtra {
   /**
    * Adds Maven resolver plugin.
    */
+  @deprecated(
+    "Noop. sbt-maven-resolver is not currently maintained and hasn't been released for sbt 1.0.",
+    "1.0.5")
```

Comment:
I think a better message (where and in the log message below) is:

> The maven resolver plugin is no longer released as a part of sbt. See https://github.com/sbt/sbt-maven-resolver.

(and then we should put some better, current info in that repo's README)

Also the version should be 1.2.0 (when this PR will probably land), or maybe (retroactively) 1.0.0.

Reply:
I guess it should be 1.0.0.